src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/IndexControllerTests.kt

```diff
@@ -252,5 +252,31 @@ class IndexControllerTests
         assertThat(result.reportDisplayNames).isNull()
     }
 
+    @Test
+    fun `builds and shows run a report button`()
```

Comment:
This name is a bit confusing - we're checking whether the controller set the permission correctly, not whether a button is shown (or not shown)